### ISSUE TITLE
PrevSlideNumber/Object not updating on SlideComplete?

### ISSUE THREAD

**Issue by pdcarroll** [iosscripts/iosslider#203]
From within the onSlideComplete callback, the 'previous' slide parameters in the args object are not updating. PrevSlideNumber is always 1, no matter which slide I'm on. 'target' and 'current' slide parameters are updating fine. Is this a bug, or am I missing something?

Thanks!

**Comment by marcwhitbread** [comment]
Probably a bug. I've had issues with these values in the past. Let me take a look and I'll back to you. 

In the mean time, a workaround. You can use a global object to store the current 'args' object and save it for the next callback and act as the previous slide 'args' object. Like so:

``` js
var prevArgs;

$('.iosSlider').iosSlider({
    onSlideComplete: function(args) {
        //current slide arguements
        args;
        //previous slide arguements
        prevArgs = args;
    }   
});
```

**Comment by pdcarroll** [comment]
Thanks for the quick response. I'll use your workaround for now.

**Comment by marcwhitbread** [comment]
Fixed in v1.1.64
